### PR TITLE
awk: update Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by gutjuri — APPROVED** [review]

File: pages.es/common/awk.md [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
+- Imprime la quinta columna (también conocida como campo) de un archivo separado por espacios:
 
-`awk '{print $5}' {{archivo}}`
+`awk '{print $5}' {{ruta/a/archivo}}`
 
-- Imprime la segunda columna de las líneas que contengan "algo" en un archivo separado por espacios:
+- Imprime la segunda columna de las líneas que contienen "foo" en un archivo separado por espacios:
 
-`awk '/{{algo}}/ {print $2}' {{archivo}}`
+`awk '/{{foo}}/ {print $2}' {{ruta/a/archivo}}`
```
Comment (kbdharun) [inline pages.es/common/awk.md:12]:
```suggestion
`awk '/{{foo}}/ {print $2}' {{ruta/al/archivo}}`
```

File: pages.es/common/awk.md [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
 
-> Un lenguaje de programación versátil para trabajar con archivos.
+> Un versátil lenguaje de programación para trabajar con archivos.
 > Más información: <https://github.com/onetrueawk/awk>.
 
-- Imprime la quinta columna (también conocido como campo) en un archivo separado por espacios:
+- Imprime la quinta columna (también conocida como campo) de un archivo separado por espacios:
 
-`awk '{print $5}' {{archivo}}`
+`awk '{print $5}' {{ruta/a/archivo}}`
```
Comment (kbdharun) [inline pages.es/common/awk.md:8]:
```suggestion
`awk '{print $5}' {{ruta/al/archivo}}`
```

File: pages.es/common/awk.md [hunk 3]
```diff
@@ -1,36 +1,36 @@
...
+- Imprime la segunda columna de las líneas que contienen "foo" en un archivo separado por espacios:
 
-`awk '/{{algo}}/ {print $2}' {{archivo}}`
+`awk '/{{foo}}/ {print $2}' {{ruta/a/archivo}}`
 
-- Imprime la última columna de cada línea de un archivo, usando la coma (en vez de espacio) como separador de campo:
+- Imprime la última columna de cada línea de un archivo, utilizando una coma (en lugar de un espacio) como separador de campos:
 
-`awk -F ',' '{print $NF}' {{archivo}}`
+`awk -F ',' '{print $NF}' {{ruta/a/archivo}}`
```
Comment (kbdharun) [inline pages.es/common/awk.md:16]:
```suggestion
`awk -F ',' '{print $NF}' {{ruta/al/archivo}}`
```

File: pages.es/common/awk.md [hunk 4]
```diff
@@ -1,36 +1,36 @@
...
 
-- Suma los valores en de la primera columna de un archivo e imprime el total:
+- Suma los valores de la primera columna de un fichero e imprime el total:
 
-`awk '{s+=$1} END {print s}' {{archivo}}`
+`awk '{s+=$1} END {print s}' {{ruta/a/archivo}}`
```
Comment (kbdharun) [inline pages.es/common/awk.md:20]:
```suggestion
`awk '{s+=$1} END {print s}' {{ruta/al/archivo}}`
```

File: pages.es/common/awk.md [hunk 5]
```diff
@@ -1,36 +1,36 @@
...
 
-- Suma los valores en de la primera columna de un archivo e imprime el total de forma bonita:
+- Imprime una de cada tres líneas a partir de la primera:
 
-`awk '{s+=$1; print $1} END {print "--------"; print s}' {{archivo}}`
+`awk 'NR%3==1' {{ruta/a/archivo}}`
```
Comment (kbdharun) [inline pages.es/common/awk.md:24]:
```suggestion
`awk 'NR%3==1' {{ruta/al/archivo}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**